src/test/java/com/codingnomads/betty/data/batch/batchwriters/FootballMatchRdsWriterTests.java

```diff
@@ -0,0 +1,42 @@
+package com.codingnomads.betty.data.batch.batchwriters;
+
+import com.codingnomads.betty.data.models.FootballMatchInfo;
+import com.codingnomads.betty.logic.interfaces.FootballMatchesInfoJpaRepository;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.mockito.Mockito.*;
+
+public class FootballMatchRdsWriterTests {
+
+    private FootballMatchesInfoJpaRepository mockFootballMatchesInfoJpaRepository;
+    private FootballMatchRdsWriter testFootballMatchRdsWriter;
+    private FootballMatchInfo mockFootballMatchInfo;
+    private List<FootballMatchInfo> matchList;
+    private List<List<FootballMatchInfo>> items;
+
+    @Before
+    public void setUp(){
+
+        mockFootballMatchesInfoJpaRepository = mock(FootballMatchesInfoJpaRepository.class);
+        testFootballMatchRdsWriter = new FootballMatchRdsWriter(mockFootballMatchesInfoJpaRepository);
+
+        mockFootballMatchInfo = mock(FootballMatchInfo.class);
+
+        matchList = Arrays.asList(mockFootballMatchInfo);
```

Comment:
Same here as above... You should use a singleton list here to prevent accidental nullpointers. Consider changing 'Arrays.asList' to 'Collections.singletonList'